src/main/java/com/orasi/bluesource/AddRoleType.java

```diff
@@ -0,0 +1,179 @@
+package com.orasi.bluesource;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import com.orasi.web.OrasiDriver;
+import com.orasi.web.webelements.Button;
+import com.orasi.web.webelements.Checkbox;
+import com.orasi.web.webelements.Element;
+import com.orasi.web.webelements.Label;
+import com.orasi.web.webelements.Link;
+import com.orasi.web.webelements.Listbox;
+import com.orasi.web.webelements.Textbox;
+import com.orasi.web.webelements.Webtable;
+import com.orasi.web.webelements.impl.internal.ElementFactory;
+
+public class AddRoleType {
+
+	private OrasiDriver driver = null;
+
+	@FindBy(linkText = "Admin") private Link lnkAdminTab;
+	@FindBy(linkText = "Role Types") private Link lnkRoleTypes;
+	@FindBy(linkText = "Add New Role Type") private Link lnkAddNewRoleTypeButton;
+	@FindBy(id = "role_type_name") private Textbox txtRoleName;
+	@FindBy(id = "role_type_billable") private Checkbox billableCheckBox;
+	@FindBy(name = "commit") private Button btnCreateRoleType;
+	@FindBy(css = "table.table-responsive.table-bordered") private Webtable rolesTable;
+	@FindBy(xpath = "//*[@id=\'new_role\']/div[1]/div/div") private Element billingLbl;
+	@FindBy(css = "table.table-striped.table-hover") private Webtable projectsTable;
+	@FindBy(xpath = "//*[@id=\"accordion\"]/div/div[13]/div/div[1]/button[1]") private Button newRoleButton;
+	@FindBy(xpath = "//*[@id=\"new_role\"]/div[1]/div/div") private Label lblBilling;
+	@FindBy(id = "role_role_type_id") private Listbox listRoleTypes;
+	@FindBy(xpath = "//input[@value='Create Role']") private Button btnAssignRoleType;
+	
+	private String newRoleTypeName;
+
+	public AddRoleType(OrasiDriver driver){
+		this.driver = driver;
+		ElementFactory.initElements(driver, this);
+	}
+
+	public void openRoleType() {
+
+		if (lnkAdminTab.isDisplayed() == true)
+		{
+			lnkAdminTab.click();
+			lnkRoleTypes.click();
+		}
+	}
+
+	public void click_AddNewRoleTypeButton() {
+
+		if (lnkAddNewRoleTypeButton.isDisplayed() == true)
+		{
+
+			lnkAddNewRoleTypeButton.click();
+		}
+	}
+
+	public void createNewRole(String roleType) {
+		
+		this.newRoleTypeName = roleType;
```

Comment:
I would avoid using a page class variable to hold data for you.  Instead I would hold that info in the test class.